uaclient/schema/status_1.py

```diff
@@ -0,0 +1,132 @@
+"""
+Schema v.1: consolidate account and contract keys.
+
+This schema version is incompatible with previous status.json schema because:
+ - account key is now a dict instead of string to collect related id, external_account_ids and name under a single object
+ - subscription-id and subscription keys are dropped
+
+General notes:
+
+   We will increment a schema version for incompatible changes:
+    - changing the value type for existing keys
+    - dropping deprecated keys
+
+   Each schema patch file will:
+    - be named <schema_version_number>.py  # schema_version_number is an int
+    - contain up() and down() functions to adapt existing status values to the
+      desired incremental or decremental schema version. 
+    - the functions up and down will return the updated schema dict
+"""
+import copy
+import re
+import yaml
+
+from uaclient.config import UAConfig
+
+from typing import Any, Dict
+
+NEW_KEYS = ("contract",)
+DROPPED_KEYS = ("subscription", "subscription-id")
+CHANGED_KEYS = ("account",)  # keys where the value type has changed
+
+PATCH_VERSION = float(re.sub(r".*status_([.\d]+).py", r"\1", __file__))
```

Comment:
I think this is amazing and wonderful and also funny how complicated the code is to avoid writing duplicating the version number between the file name and this variable :)